collect_app/src/main/res/layout/submission_summary_layout.xml

```diff
@@ -0,0 +1,70 @@
+<LinearLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/margin_large"
+    android:background="?colorSurface"
+    android:orientation="vertical"
+    android:paddingHorizontal="@dimen/margin_standard"
+    android:paddingVertical="@dimen/margin_small"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <TextView
+            android:id="@+id/submission_name"
+            style="@style/TextAppearance.AppCompat.Headline"
```

Comment:
This should use of our Material Theme text appearances (`TextAppearance.Collect.X`). I'm guessing `Headline6` is too big but have a play with it!

Reply:
Headline6 looks pretty good. Done.